library/gen-isa-tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
```

Comment:
It could be cool to use `/bin/sh` here, and don't rely on Bash. This would make the script more portable.

Reply:
When switching to `sh`, there's stuff to fix because the script is using bash specific features. And I don't want to spend time fixing this currently... (PR against my PR welcome :p)